skops/io/_visualize.py

```diff
@@ -168,7 +168,7 @@ def pretty_print_tree(
 
 
 def walk_tree(
-    node: Node | dict[str, Node] | list[Node],
+    node: VALID_NODE_CHILD_TYPES | dict[str, VALID_NODE_CHILD_TYPES],
```

Comment:
isn't `dict(str, Node)` included in `VALID_NODE_CHILD_TYPES`?

Reply:
Yes, but this is `dict[str, VALID_NODE_CHILD_TYPES]`, so it could be something where the key is not a `Node`, like `{"foo": None}`